app/src/utils/submission.ts

```diff
@@ -0,0 +1,70 @@
+// Note: This util is meant to run serverside.
```

Comment:
Notes for reviewers: This utility handles translating our `session` object into the shape that the mock API expects.